sync/pacman-script

```diff
@@ -40,3 +41,18 @@ for codebase in $(yq eval -o=j $YAMLFILE | jq -cr '.[]'); do
     echo "Moving '${name}' from '${temporaryExtraction}' to '${targetDestination}'";
     cp -fa "${temporaryExtraction}" "${targetDestination}"
 done
+
+# remove any not needed files
+for codebase in $(yq eval -o=j $YAMLFILE | jq -cr '.[]'); do
+    name=$(echo $codebase | jq -r '.name' -)
+    destination=$(echo $codebase | jq -r '.destination' -)
+    remove=$(echo $codebase | jq -r '.remove' -)
+    if [ "$remove" = "null" ]; then
+        continue
+    fi
```

Comment:
Is it possible for `remove` to not be `"null"` exactly (e.g. an empty list or a string)? Perhaps continuing if `remove` is not a list would be better?

Reply:
This should only ever be null or a list.
Checking if it is not a list is probably harder / would require another call to jq rather than a simple string check